bundles/org.openhab.binding.paradoxalarm/src/main/java/org/openhab/binding/paradoxalarm/internal/communication/CommunicationState.java

```diff
@@ -368,7 +367,6 @@ protected void runPhase(IParadoxInitialLoginCommunicator communicator, Object...
             if (communicator != null) {
                 communicator.close();
             }
-            ParadoxPanel.getInstance().dispose();
```

Comment:
Verify that if the communicator wents offline and then online the lack of disposal of internal collections (partitions and zones) will not make any side effect when reconnecting. You can test that with the RESET or LOGOUT functionality.
If that's fine, we can get rid of the dispose() method completely in the ParadoxPanel class.

Reply:
I think the best way to dispose the panel is in ParadoxIP150BridgeHandler.dispose() which calls CommunicationState.logout(communicator) and, according to comment:
```
// For some reason after sending logout packet the connection gets reset from the other end
// currently workaround is to run directly offline phase, i.e. close socket from our end
```
I'll do some tests.

Reply:
I've done more tests and I discovered that without  ParadoxPanel.dispose(), if you disable and enable bridge thing, bridge won't get ONLINE again but if we dispose the Panel in ParadoxIP150BridgeHandler.dispose() as mentioned above, everything works fine.

I've pushed the change.